src/main/java/org/schabi/newpipe/extractor/services/soundcloud/SoundcloudStreamExtractor.java

```diff
@@ -150,6 +147,16 @@ public String getDashMpdUrl() {
         return null;
     }
 
+    @Override
+    public List<Subtitles> getSubtitlesDefault() throws IOException, ExtractionException, JsonParserException {
+        return null;
+    }
+
+    @Override
+    public List<Subtitles> getSubtitles(SubtitlesFormat format) throws IOException, ExtractionException, JsonParserException {
```

Comment:
Please don't throw a JsonParserException. Please only throw IOException or ExtractionException.
Catch the JasonParserException and throw a ExtractionException embedding the JsonParserException.

Reply:
Should be OK now.